docs/getting_started/integrating_into_django.md

```diff
@@ -54,6 +54,12 @@ MEDIA_ROOT = os.path.join(BASE_DIR, 'media')
 MEDIA_URL = '/media/'
 ```
 
+Set the `DATA_UPLOAD_MAX_NUMBER_FIELDS` setting to 10,000 or higher. This specifies the maximum number of fields allowed in a form submission, and it is recommended to increase this from Django's default of 1,000, as particularly complex page models can exceed this limit within Wagtail's page editor:
```

Comment:
Do the docs use commas for the thousand separator? Some languages could read it as `10.000` and be confused.

```suggestion
Set the `DATA_UPLOAD_MAX_NUMBER_FIELDS` setting to 10000 or higher. This specifies the maximum number of fields allowed in a form submission, and it is recommended to increase this from Django's default of 1000, as particularly complex page models can exceed this limit within Wagtail's page editor:
```

Reply:
Sure, happy to go along with that!